mysql-test/suite/wsrep/t/wsrep-recover.test

```diff
@@ -0,0 +1,75 @@
+#
```

Comment:
If I understand correctly, this test won't exercise the case where there are gaps in XIDs, as detected by `wsrep_order_and_check_continuity`. Would it make sense to create such a test?